nucliadb_sdk/nucliadb_sdk/v2/sdk.py

```diff
@@ -202,9 +202,14 @@ def _func(self: "NucliaDB", content: Optional[Any] = None, **kwargs):
             raise TypeError(f"Invalid arguments provided: {kwargs}")
 
         if not stream_response:
-            resp = self._request(
-                path, method, data=data, query_params=query_params, content=raw_content
-            )
+            if isinstance(self, NucliaDBAsync):
```

Comment:
I think the real problem is that the `_request` method of the `NucliaDBAsync` class is missing the `content` parameter. See: https://github.com/nuclia/nucliadb/blob/main/nucliadb_sdk/nucliadb_sdk/v2/sdk.py#L843-L849

Can you add it? Then you don't need this conditional check

Reply:
Thanks for the review @lferran. I have implemented your suggested change. I feel like most of the logic should be moved to the base class instead, but as a quick fix I think this is good enough